src/includes/closures/Shared/CachePathUtils.php

```diff
@@ -2,6 +2,39 @@
 namespace WebSharks\ZenCache\Pro;
 
 /*
+* Cache-path suffix frag (regex).
+*
+* @since 15xxxx Enhancing translation support.
+*
+* @param string $regex_suffix_frag Existing regex suffix frag?
+*
+* @return string Cache-path suffix frag (regex).
+*/
+$self->cachePathRegexSuffixFrag = function ($regex_suffix_frag = CACHE_PATH_REGEX_DEFAULT_SUFFIX_FRAG) use ($self) {
+    if ($regex_suffix_frag === CACHE_PATH_REGEX_DEFAULT_SUFFIX_FRAG) {
+        return $self->cachePathRegexDefaultSuffixFrag();
+    }
+    return (string) $regex_suffix_frag;
+};
+
+/*
+* Default cache-path suffix frag (regex).
+*
+* @since 15xxxx Enhancing translation support.
+*
+* @return string Default cache-path suffix frag (regex).
+*/
+$self->cachePathRegexDefaultSuffixFrag = function () use ($self) {
+    if ($self->isPlugin() && !empty($GLOBALS['wp_rewrite'])){
```

Comment:
I think it's fine. `$GLOBALS['wp_rewrite']` refers to an instance of the `WP_Rewrite` class, and those pagination values are both properties of that class; i.e., expected to always exist.